tests/test_modeling_transfo_xl.py

```diff
@@ -232,8 +232,10 @@ def test_retain_grad_hidden_states_attentions(self):
         return
 
     @require_torch_multi_gpu
+    @unittest.skip(
```

Comment:
changed all other model's that skip DP to unittest.skip -> should be easier to parse to a table later